src/bot/cogs/moderator.py

```
@@ -354,7 +356,10 @@ async def send_moderator_log(self, guild: discord.Guild, moderator: discord.User
         await log_channel.send(embed=embed)
 
     async def send_mod_log(self, guild: discord.Guild, user: discord.User, action: str, reason: str, message_content: str = ""):
-        basic_setting = await self.bot.api.get_moderator_settings(str(guild.id))
+        try:
+            basic_setting = await self.bot.api.get_moderator_settings(str(guild.id))
+        except:
```

Ruff (0.15.7)

[error] Do not use bare `except`

(E722)